apps/omg_watcher_info/lib/omg_watcher_info/db/block.ex

```diff
@@ -92,6 +92,56 @@ defmodule OMG.WatcherInfo.DB.Block do
     |> Paginator.set_data(paginator)
   end
 
+  @spec query_last_24_hour(Ecto.Query.t()) :: Ecto.Query.t()
+  def query_last_24_hour(query) do
```

Comment:
This is `DB` level functions should be as generic as possible. E.g.

```elixir
defmodule OMG.WatcherInfo.DB.Block do
  def query_timestamp_between(start_datetime, end_datetime) do
    # ...
  end
end

```

Then we have `OMG.WatcherInfo.API.Stats` module doing:

```elixir
end_datetime = DateTime.to_unix(DateTime.utc_now())
start_datetime = end_datetime - @seconds_in_twenty_four_hours

DB.Block.all_by_timestamp_between(start_datetime, end_datetime)
```

Reply:
Done 8946c62.